### PR TITLE
Update/1.23.0

### PR DESCRIPTION
**Pull request by ELundby45**
# PyCairo v1.23.0

upstream: https://github.com/pygobject/pycairo/tree/v1.23.0
`meson.build`: https://github.com/pygobject/pycairo/blob/v1.23.0/meson.build
changelog: https://pycairo.readthedocs.io/en/latest/changelog.html#v1-23-0

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,15 +31,15 @@ requirements:
...
+    - cairo 1.16.0
+    - flake8 6.0.0 # for testing during the build
+    - glib {{glib}}
+    - pytest 7.1.2 # for testing during the build
     - python
     - setuptools
     - wheel
   run:
-    - cairo
+    - cairo >=1.15.10
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - cairo >=1.16.0,2.0a0
```

**Review by cbouss — COMMENTED** [review]
There is also an abs.yaml in the feedstock that can be removed.

File: recipe/meta.yaml (hunk 1 above)
Comment (ELundby45) [reply recipe/meta.yaml]: https://github.com/pygobject/pycairo/blob/v1.23.0/meson.build#L12
Has `cair_version_req = '>=1.15.10'` can I ask where you see `>=1.16.0,2.0a0` ?

Comment (cbouss) [reply recipe/meta.yaml]: Just because our cbc has version 1.16.0 and cairo has a run export on the major version.

Comment (ELundby45) [reply recipe/meta.yaml]: Ahah. Thanks! Didn't even think to check cbc for cairo

**Review by cbouss — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**